cmd/run.go

```diff
@@ -331,11 +331,13 @@ func saveHeartbeats(v *viper.Viper) {
 func sendDiagnostics(v *viper.Viper, d diagnostics) error {
 	paramAPI, err := params.LoadAPIParams(v)
 	if err != nil {
-		var errauth api.ErrAuth
+		// Prevent sending diags for api key errors
```

Comment:
Jus to be consice with other comments.
```suggestion
		// Prevent sending diags for api key errors.
```